### PR TITLE
KEYCLOAK-12962 Enforce 3.6.0 maven version for deploy phase

### PR DESCRIPTION
**Pull request by mhajas**

### CODE REVIEW

File: testsuite/integration-arquillian/pom.xml [hunk 1]
```diff
@@ -309,6 +309,20 @@
                             </rules>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>enforce-maven</id>
```
Comment (vramik) [inline testsuite/integration-arquillian/pom.xml]: nit: shouldn't it be different from other execution ids? what about `enforce-maven-deploy`? But it's rather optional, feel free to ignore.

**Review by vramik — COMMENTED** [review]
@mhajas thank you, I have just small nit.

File: testsuite/integration-arquillian/pom.xml (hunk 1 above)
Comment (martin-kanis) [reply testsuite/integration-arquillian/pom.xml]: +1

**Review by martin-kanis — CHANGES_REQUESTED** [review]

File: testsuite/integration-arquillian/pom.xml (hunk 1 above)
Comment (mhajas) [reply testsuite/integration-arquillian/pom.xml]: done

**Review by martin-kanis — APPROVED** [review]

**Review by vramik — APPROVED** [review]

**Comment by hmlnarik** [comment]
Need to close and reopen due to travis

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**